components/button/themes/spectrum-two.css

```diff
@@ -51,16 +51,17 @@
 
 			&:not(.spectrum-Button--outline) {
 				--mod-button-background-color-hover: var(--spectrum-gray-200);
-				--mod-button-background-color-down: var(--spectrum-gray-400);
+				--mod-button-background-color-down: var(--spectrum-gray-200);
 				--mod-button-background-color-focus: var(--spectrum-gray-200);
 			}
 
 			&.spectrum-Button--outline {
-				--mod-button-background-color-hover: var(--spectrum-gray-200);
-				--mod-button-background-color-down: var(--spectrum-gray-400);
-				--mod-button-background-color-focus: var(--spectrum-gray-200);
+				--mod-button-background-color-hover: var(--spectrum-gray-100);
+				--mod-button-background-color-down: var(--spectrum-gray-100);
+				--mod-button-background-color-focus: var(--spectrum-gray-100);
 
-				--mod-button-border-color-default: var(--spectrum-gray-200);
+				--mod-button-border-color-default: var(--spectrum-gray-300);
+				--mod-button-border-color-down: var(--spectrum-gray-400);
```

Comment:
Do we need to add a different fallback value for spectrum 1?

Reply:
Great point, I didn't realize the down states didn't have a defined color in S1, I've added this.